fuzz/fuzz_targets/decompress_lzma_stream.rs

```diff
@@ -0,0 +1,40 @@
+#![no_main]
+#[macro_use]
+extern crate libfuzzer_sys;
+
+use lzma_rs::error::Result;
+use std::io::Write;
+
+fn decode_lzma(compressed: &[u8]) -> Result<Vec<u8>> {
+    let mut bf = std::io::Cursor::new(compressed);
```

Comment:
Nit: `bf` is not really a clear name. I'd suggest just importing `std::io::Cursor` in this file, and directly inline this without giving it a name, i.e. `lzma_decompress(&mut Cursor(compressed), ...)`.